cache/src/main/scala/org/allenai/common/cache/JsonQueryCache.scala

```diff
@@ -25,7 +41,10 @@ class JsonQueryCache[V: JsonFormat] @Inject() (
     * become part of the cache key (prepended to the actual query)
     */
   def this(redisHostName: String, redisHostPort: Int, redisTimeout: Int, clientPrefix: String) =
-    this(new JedisPool(new JedisPoolConfig, redisHostName, redisHostPort, redisTimeout), clientPrefix)
+    this(
+      new JedisPool(new JedisPoolConfig, redisHostName, redisHostPort, redisTimeout),
```

Comment:
I've been resisting the urge to remove these in case somebody's using them in another project, but I always forget that their code won't break unless they explicitly bump the version number.